open-ce/build_env.py

```diff
@@ -72,8 +73,11 @@ def _all_outputs_exist(output_folder, output_files):
     return all([os.path.exists(os.path.join(os.path.abspath(output_folder), package))
                     for package in output_files])
 
-def build_env(args):
+def build_env(args): #pylint: disable=too-many-branches
     '''Entry Function'''
+    if utils.is_url(args.conda_build_config):
+        args.conda_build_config, _ = urllib.request.urlretrieve(args.conda_build_config)
```

Comment:
I think we need to add try block here.

Reply:
I think the current try blocks we have give a useful error if something goes wrong:
```
$ open-ce/open-ce build env https://raw.githubusercontent.com/bethune-bryant/open-ce-environments/test_branch/envs/test-env.yam --skip_build_packages --repository_folder /tmp/bryant/test
[OPEN-CE-ERROR-10] Error creating Build Tree
[OPEN-CE-ERROR-0] Unexpected Error: Error in https://raw.githubusercontent.com/bethune-bryant/open-ce-environments/test_branch/envs/test-env.yam:
  HTTP Error 404: Not Found
```

Reply:
I did go ahead and add exception handling. The error now looks like:
```
$ open-ce/open-ce build env https://raw.githubusercontent.com/bethune-bryant/open-ce-environments/test_branch/envs/test-env.yaml --skip_build_packages --repository_folder /tmp/bryant/test --conda_build_config https://doesnotexisttest.com/cond_build_config.yaml
[OPEN-CE-ERROR-24] Failed to download https://doesnotexisttest.com/cond_build_config.yaml with error:
<urlopen error [Errno -2] Name or service not known>
```